src/main/java/org/folio/circulation/services/feefine/AccountCancelCommand.java

```diff
@@ -0,0 +1,15 @@
+package org.folio.circulation.services.feefine;
+
+import org.folio.circulation.domain.AccountCancelReason;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+public final class AccountCancelCommand {
```

Comment:
I think commands are well suited to using imperative language, to me `cancel account` reads better than `account cancel`. What do you think?

```suggestion
public final class CancelAccountCommand {
```

Reply:
Agreed, renamed.